extensions/exposition/source/HTTP/Context.ts

```diff
@@ -0,0 +1,60 @@
+import Negotiator from 'negotiator'
+import { Timing } from './Timing'
+import { type Format, formats, types } from './formats'
+import { read } from './messages'
+import type { OutgoingMessage } from './messages'
+import type * as http from 'node:http'
+
+export class Context {
+  public readonly request: IncomingMessage
```

Comment:
As description says that this PR is a step toward HTTP/2 adoption I believe it will be wise to make type here more future proof as an intersection:
```ts
public readonly request: http.IncomingMessage & http2.Http2ServerRequest
```

so we will not unintentionally use something outside of [compatibility API](https://nodejs.org/api/http2.html#compatibility-api) and make future refactoring easier.

Reply:
As discussed offline - let's do it as a separated effort or as part of HTTP/2 actual implementation.